src/deploy_job.rs

```diff
@@ -45,7 +45,7 @@ impl DeployJob {
             Self::env_var("ARGO_SUBMIT", "true"),
             Self::env_var("WORKFLOW_GENERATOR_PROJECT_NAME", &gordo.metadata.name),
             Self::env_var("WORKFLOW_GENERATOR_OWNER_REFERENCES", &owner_ref_as_string),
-            Self::env_var("WORKFLOW_GENERATOR_PROJECT_VERSION", &project_revision),
+            Self::env_var("WORKFLOW_GENERATOR_PROJECT_REVISION", &project_revision),
         ];
```

Comment:
When you have the backward compability logic for models, does it not make sense to have something similar here, and set both the old and new env variable?

Reply:
This is why we need you. 👀